app/AcquireIntersonArrayRF.cxx

```diff
@@ -127,12 +127,18 @@ int main( int argc, char * argv[] )
   std::cout << "Max RF samples: " << width_samples << std::endl;
   const itk::SizeValueType framesToCollect = frames;
 
+  const double ns = ContainerType::MAX_RFSAMPLES; // number of samples
+  const double fs = frequencies[ frequencyIndex ] / 1000; // [kHz]=[samples/ms] - frequency per sample
+  double depth = c * ns / ( 2 * fs );
+  std::cout << "Depth: " << depth << "mm" << std::endl;
+
   container.AbortScan();
   container.SetRFData( true );
 
-  container.IdleInitScanConverter( depth, width_samples, height_lines, probeId,
+  const int depth_acq = 100; // Depth of acquisition [mm]
+  container.IdleInitScanConverter( depth_acq, width_samples, height_lines, probeId,
       steering, false, false, 0 );
-  container.HardInitScanConverter( depth, width_samples, height_lines, steering );
+  container.HardInitScanConverter( depth_acq, width_samples, height_lines, steering );
```

Comment:
I am still not certain this is right.  They have a "mmperpixel" function, and the value returned by that function changes when depth changes, and the bmode image that is returned changes when this depth value changes.